package.json

```diff
@@ -29,61 +29,54 @@
     "lint:fix-js": "npm run lint:js -- --fix",
     "lint:scss": "stylelint ./src/scss/**/*",
     "lint:fix-scss": "npm run lint:scss -- --fix",
-    "format": "prettier --write **/*"
+    "format": "prettier --write **/*",
+    "prepare": "husky install",
+    "lint-staged": "lint-staged"
   },
   "ava": {
     "babel": false
   },
   "lint-staged": {
     "*.{js,css,json,md}": [
-      "prettier --write",
-      "git add"
+      "prettier --write"
     ],
     "*.js": [
-      "eslint --fix",
-      "git add"
+      "eslint --fix"
     ],
     "*.scss": [
-      "stylelint --fix",
-      "git add"
+      "stylelint --fix"
     ]
   },
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged"
-    }
-  },
   "dependencies": {
     "@upstatement/puppy": "^0.10.0"
```

Comment:
Maybe we should wait until the puppy-lib PR is merged and bump this version in this PR

Reply:
Makes sense to me!